scholly/base/src/layouts/layout3/components/layout/HeaderData.js

```diff
@@ -15,6 +15,10 @@ const HeaderData = () => [
     title: 'Team Members',
     link: '/team',
   },
+  {
+    title: 'Awards',
```

Comment:
So are we calling it `Awards` in the UI, but internally we're referring to it as "achievements." It'd be good to maintain consistency IMO

Reply:
The file has been deleted as it was unused.